### PR TITLE
fix: center button and toolbar text with relevant icons

### PR DESCRIPTION
**Pull request by JFrankfurt**
## before
image

## after
image

### CODE REVIEW

File: src/components/Swap/Toolbar/Caption.tsx [hunk 1]
```diff
@@ -27,8 +27,8 @@ interface CaptionProps {
 function Caption({ icon: Icon = AlertTriangle, caption }: CaptionProps) {
   return (
     <>
-      <Icon color="secondary" />
-      <ThemedText.Caption>{caption}</ThemedText.Caption>
+      <Icon color="secondary" size={16.67 / 14} />
```
Comment (zzmp) [inline src/components/Swap/Toolbar/Caption.tsx:30]: Is this sizing necessary?

File: src/components/Swap/Toolbar/Caption.tsx [hunk 2]
```diff
@@ -27,8 +27,8 @@ interface CaptionProps {
...
+      <ThemedText.Caption lineHeight="14px">{caption}</ThemedText.Caption>
```
Comment (zzmp) [inline src/components/Swap/Toolbar/Caption.tsx:31]: I thought we were leaving this until post-phase0.

Comment (JFrankfurt) [reply src/components/Swap/Toolbar/Caption.tsx:31]: no, just the font-weight

alignment is going in

File: src/components/Swap/Toolbar/Caption.tsx (hunk 1 above)
Comment (JFrankfurt) [reply src/components/Swap/Toolbar/Caption.tsx:30]: it is... but it looks like it's not actually being applied

Comment (JFrankfurt) [reply src/components/Swap/Toolbar/Caption.tsx:30]: was working with the old `largeiconcss`, but that's not here now

**Review by cmcewen — APPROVED** [review]

**Pull request merged**